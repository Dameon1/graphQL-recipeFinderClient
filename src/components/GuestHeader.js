import React from "react";
import { Link } from "@reach/router";
import { NavButton } from "./styles";

export default class GuestHeader extends React.Component {
  render() {
    return (
      <nav>
        <Link to="/signIn">
          <NavButton>Login</NavButton>
        </Link>
        <Link to="/signUp">
          <NavButton>Sign up</NavButton>
        </Link>
        <Link to="/dashboard">
          <NavButton>Dashboard</NavButton>
        </Link>
        <Link to="/">
          <NavButton>Home</NavButton>
        </Link>
      </nav>
    );
  }
}
