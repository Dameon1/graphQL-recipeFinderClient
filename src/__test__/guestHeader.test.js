import React from "react";
import { shallow, mount } from "enzyme";
import GuestHeader from "../components/GuestHeader";
import toJSON from "enzyme-to-json";

describe("<GuestHeader />", () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<GuestHeader />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});