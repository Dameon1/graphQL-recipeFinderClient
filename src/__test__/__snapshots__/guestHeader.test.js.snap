// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuestHeader /> matches the snapshot 1`] = `
<nav>
  <ForwardRef
    to="/signIn"
  >
    <styled.button>
      Login
    </styled.button>
  </ForwardRef>
  <ForwardRef
    to="/signUp"
  >
    <styled.button>
      Sign up
    </styled.button>
  </ForwardRef>
  <ForwardRef
    to="/dashboard"
  >
    <styled.button>
      Dashboard
    </styled.button>
  </ForwardRef>
  <ForwardRef
    to="/graphQL-recipeFinderClient"
  >
    <styled.button>
      Home
    </styled.button>
  </ForwardRef>
</nav>
`;
